lcls-twincat-optics/lcls_twincat_optics_plc/POUs/Main.TcPOU

```diff
@@ -24,35 +26,22 @@ VAR
 END_VAR
 ]]></Declaration>
     <Implementation>
-      <ST><![CDATA[(*
-TestPitch.Stepper.bLimitBackwardEnable;
-TestPitch.Stepper.bLimitForwardEnable;
-TestPitch.Stepper.bHardwareEnable;
-TestPitch.Stepper.fVelocity := 150.0;
+      <ST><![CDATA[M1.bLimitBackwardEnable;
+M1.bLimitForwardEnable;
+M1.bHardwareEnable;
```

Comment:
Are these just to make the values visible while testing?

Reply:
Yes and to write to as well. Everything in Main is just used for testing